internal/app/app.go

```diff
@@ -79,11 +79,18 @@ type FileChange struct {
 	Patch       string `json:"patch"`
 }
 
+// PRReviewMap is a map of file paths to a list of PRCommentInfo
 type PRReviewMap map[string][]PRCommentInfo
 type PRCommentInfo struct {
 	CommentBody string
-	StartLine   int
-	Line        int
+	// The start_line is the first line in the pull request diff that your multi-line comment applies to.
+	StartLine int
+	// The line of the blob in the pull request diff that the comment applies to. For a multi-line comment, the last line of the range that your comment applies to.
```

Comment:
It is good practice to include documentation for newly introduced properties in a struct. This helps other developers understand the purpose and usage of these properties.